usagov/usagov-2021
# usagov-1840-cypress-secondary-nav-page-test: fix for hover background…

… color validation


## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1480

## Description

Fix hover background color validation for the content list of inks in the body
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


Run the script secondary_nav_page_2024.cy.js
Verify that all test pass in English and Spanish
<img width="1430" alt="Screenshot 2024-08-29 at 1 45 10 PM" src="https://github.com/user-attachments/assets/e7b9909d-9df2-4c41-a558-bce35315098f">

 
Verify the first test passes the hover color validation
<img width="1155" alt="Screenshot 2024-08-28 at 3 32 25 PM" src="https://github.com/user-attachments/assets/cf9c2d55-1636-4e38-8823-e3c82c4dc4ef">

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
